packages/app/src/cli/services/dev/ui/components/DevSessionUI.tsx

```diff
@@ -207,6 +183,91 @@ const DevSessionUI: FunctionComponent<DevSesionUIProps> = ({
               </>
             )}
           </Box>
+        </>
+      ),
+    },
+    // eslint-disable-next-line id-length
+    a: {
+      label: 'App info',
+      content: (
+        <Box flexDirection="column">
+          <TabularData
+            tabularData={[
+              ['App:', appName ?? ''],
+              ['App URL:', appURL ?? ''],
+              ['Config:', configPath?.split('/').pop() ?? ''],
```

Comment:
Would it be useful to show the full path (`/Users/Nick/tests/app2/shopify.app.toml`) instead of the file name?

Reply:
Elsewhere we use just the file name referencing the config, since they are always in the app root. So I think this is OK and less noisy.